app/templates/main/browse.html

```diff
@@ -335,59 +335,56 @@ <h1 class="govuk-heading-l browse__records-found__text">Records found {{ num_rec
+                <dt class="govuk-summary-list__key govuk-summary-list__key__left">
                   Closure period
                 </dt>
               </div>
 
               <!-- Table Content -->
               {% for record in results %}
-                <div class="govuk-summary-list__row">
-                  <dd class="govuk-summary-list__value browse__table__large-width">
-                    <a href="{{url_for('main.browse', transferring_body_id=record['transferring_body_id']) }}">
-                      {{ record["transferring_body"] }}
+              <div class="govuk-summary-list__row">
+                <dd class="govuk-summary-list__value">
+                    {{ record[2]|format_datetime}}
+                </dd>
+                <dd class="govuk-summary-list__value">
+                    <a href="{{ url_for('main.record', record_id=record[0]) }}">
+                      {{ record[1] }}
                     </a>
-                  </dd>
-                  <dd class="govuk-summary-list__value">
-                    <p class="govuk-body browse__body">
-                      <a href="{{ url_for('main.browse', series_id=record['series_id']) }}">
-                        {{ record["series"] }}
-                      </a></p>
-                  </dd>
-                  <dd class="govuk-summary-list__value">
-                    <p class="govuk-body browse__body browse__table__right-align">{{ record["last_record_transferred"] }}</p>
-                  </dd>
-                  <dd class="govuk-summary-list__value">
-                    <p class="govuk-body browse__body browse__table__right-align">{{ record["records_held"] }}</p>
-                  </dd>
-                  <dd class="govuk-summary-list__value">
-                    <p class="govuk-body browse__body browse__table__right-align">{{ record["consignment_in_series"] }}</p>
-                  </dd>
-                </div>
+                </dd>
+                <dd class="govuk-summary-list__value">
+                  <strong class="govuk-tag {{ 'govuk-tag--green' if record[3] == 'Open' else 'govuk-tag--red' }}">
+                    {{ record[3] }}
+                  </strong>
+                </dd>
+                <dd class="govuk-summary-list__value">
+                  {{ record[4]|format_datetime}}
+                </dd>
+                <dd class="govuk-summary-list__value">
+                  <p class="govuk-body browse__body">{% if record[5] %} {{ record[5] }} years {% else %} - {% endif %}</p>
+                </dd>
+              </div>
               {% endfor %}
-          </dl>
+            </dl>
           {% endif %}
         {% endif %}
       {% endif %}
 
-      {% if num_records_found > 0 %}
-        <!-- PAGINATION -->
-        {% with view_name='main.browse' %}
-          {% include "pagination.html" %}
-        {% endwith %}
-      {% endif %}
+      <!-- PAGINATION -->
```

Comment:
pagination tremplate has condition of total pages > 1 inside it anyway so this condition was redundant